src/App.tsx

```diff
@@ -17,18 +77,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onChangeInput={handleInput}
+                query={query}
+                onClearInput={handleClearInput}
+                selectedOption={selectedStatus}
+                onSelectedStatus={setSelectedStatus}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length
+                ? (
+                  <TodoList
+                    todos={vissibleTodos}
+                    onClickUser={handleUserInfo}
+                    isClickedTodoInfo={isClickedTodoInfo}
+                  />
+                )
+                : <Loader />}
```

Comment:
Try to render your loader with other options. For instance, todos.length === 0 && Loader. It should help you to pass the test with loader

Reply:
done